Spanner/src/ValueMapper.php

```diff
@@ -324,6 +327,7 @@ private function decodeValue($value, array $type)
                 }
                 break;
 
+            case self::TYPE_FLOAT32:
```

Comment:
While passing the `FLOAT32` to the Spanner server, we keep the type as `FLOAT64`? If yes, could you please give me some reference to this requirement?

Reply:
Sorry, I'm a bit confused -- isn't this code for decodeValue, which is triggered when the values are received from the server?

Reply:
Yeah, this is decoding, I was talking about encoding here. In the discussion we had offline, I understand we don't need to encode the value as `FLOAT32` as the table's columns are already typed and Server will take care of the value - type mappings.